spec/restaurant_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'restaurant'
+
+describe Restaurant do
+
+  it 'has a menu of dishes with prices' do
+    expect(subject.menu).to eq({margherita: 5, pepperoni: 6, hawaiian: 6, supreme: 7})
+  end
+
+  it 'allows to choose one or more of the same dishes' do
+    expect(subject).to respond_to(:choose).with(2).arguments
+  end
+
+  it 'stores the choice the right number of times' do
+    subject.choose(2, "margherita")
+    expect(subject.selected).to eq({margherita: 2})
+  end
+
+  it 'raises an error if the selected dish is not on the menu' do
+    expect{subject.choose(1, "banana")}.to raise_error "Your choice is not on the menu"
```

Comment:
Line is too long. [87/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.